File: Iniciante/1165.py
def ehPrimo(x):
  for i in range(2, x):
    if (x%i == 0):
      return False

  return True

n = int(input())

for i in range(n):
  x = int(input())
  
  if (ehPrimo(x)):
    print("{} eh primo".format(x))
  else:
    print("{} nao eh primo".format(x))

File: Iniciante/1165.js
function ehPrimo(x) {
  for (let i = 2; i < x; i++) {
    if (x%i == 0) {
      return false;
    }
  }
  return true;
}

let n = Number(lines.shift());

for (let i = 0; i < n; i++) {
  let x = Number(lines.shift());
  
  if (ehPrimo(x)) {
    console.log(`${x} eh primo`);
  } else {
    console.log(`${x} nao eh primo`);
  }
}
